aqw-opcua-server/Settings.h

```diff
@@ -29,24 +29,20 @@ namespace weathersvr {
 
         private:
 
-            void setDefaultValues();
             /*
             Check if the values present in the settings.json file related to the dark sky api are valid before set them to respective variables. If is not valid, the default values will be kept.
             This function may throw an exception web::json::json_exception.
-            @param web::json::value& jsonObj - The Json OBJECT which the values will be parsed and validated.
+            @param web::json::value& jsonObj - The Json OBJECT which values will be parsed and validated.
             */
-            void validateValuesFromDarkSky(web::json::value& jsonObj);
+            bool validateValuesFromDarkSky(web::json::value& jsonObj);
 
             utility::string_t keyApiDarksky;
             utility::string_t units;
-            short intervalDownloadWeatherData;
+            int intervalWeatherDataDownload;
```

Comment:
Suggested `int` because `short` is used only in some specific cases. Look at this for example: https://stackoverflow.com/questions/24371077/when-to-use-short-over-int